theano/gpuarray/tests/test_reduction.py

```diff
@@ -14,7 +14,7 @@
 import math
 
 # Number of values to be used in test tensors (except with 0-D tensors!).
-test_size = 10000000
+test_size = 10000
```

Comment:
After discussion with @abergeron this is the right fix. It don't lower the quality of the test. The problem is that our reduction with axis =1 use only 1 thread on the GPU (so it isn't used by default yet. We still work to optimize that case). So this is super slow and it bust the 5s limit on windows per kernel call.